modular_zubbers/code/modules/taur_mechanics/serpentine_taur.dm

```diff
@@ -0,0 +1,11 @@
+/obj/item/organ/external/taur_body/serpentine/on_mob_insert(mob/living/carbon/organ_owner, special, movement_flags)
+	. = ..()
+
+	RemoveElement(/datum/element/footstep, FOOTSTEP_MOB_HUMAN, 0.6, -6)
```

Comment:
tbh i dont know what these args are for and i dont care

Reply:
Remove element just takes the default argument so this should be `RemoveElement(/datum/element/footstep)`

What makes me happy is that I didn't approve whatever this code was originally